docs/usage.md

```diff
@@ -157,51 +176,57 @@ The simulation mode has some sample data that helps you get a better understandi
 
 ## Production Mode
 
-1. Set global variable `Demo_mode` to *`true`*. Refer to the section [Global Variables](https://docs.fortinet.com/document/fortisoar/7.4.3/playbooks-guide/488685/dynamic-values#Global_Variables) in the **Playbooks Guide** of the platform documentation for more information.</li>
+1. Set global variable `Demo_mode` to *`false`*. Refer to the section [Global Variables](https://docs.fortinet.com/document/fortisoar/7.4.3/playbooks-guide/488685/dynamic-values#Global_Variables) in the **Playbooks Guide** of the platform documentation for more information.</li>

```

Comment:
On line number 168 add two points:
1. Does Patch Exist?</strong> as <strong><em>Unknown</em>.
2. Reset the scenario before moving in production mode.
@cs-abhishek-shukla